### PR TITLE
Feature/json representation

### PR DESCRIPTION
**Pull request by joanromano**

### CODE REVIEW

**Comment by joanromano** [comment]
@MP0w might be good to start having a look

File: KakapoExample/KakapoExample/KakapoPlayground.playground/Contents.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
+//: Playground - noun: a place where people can play
+
```
Comment (MP0w) [inline KakapoExample/KakapoExample/KakapoPlayground.playground/Contents.swift:2]: maybe we should not commit the content of the playground

File: KakapoExample/KakapoTests/KakapoServerTests.swift [hunk 2]
```diff
@@ -16,26 +16,29 @@ import Nimble
...
-        describe("#KakapoServer") {
-            
-            beforeEach({
-                KakapoServer.enable()
-            })
-            
-            afterEach({ 
-                KakapoServer.disable()
-            })
+        beforeEach({
```
Comment (MP0w) [inline KakapoExample/KakapoTests/KakapoServerTests.swift]: trailing closure

File: KakapoExample/KakapoTests/KakapoServerTests.swift [hunk 3]
```diff
@@ -16,26 +16,29 @@ import Nimble
...
+        describe("Registering urls") {
             
             it("should call the handler when requesting a registered url") {
                 var info: URLInfo? = nil
                 var responseURL: NSURL? = nil
                 
-                KakapoServer.get("/users/:id") { request in
+                KakapoServer.get("/users/:id"){ request in
                     info = request.info
+                    return db.find(UserFactory.self, id: 2)
```
Comment (MP0w) [inline KakapoExample/KakapoTests/KakapoServerTests.swift]: shouldn't it use the param passed?

File: KakapoExample/KakapoTests/KakapoServerTests.swift [hunk 4]
```diff
@@ -52,22 +55,27 @@ class KakapoServerTests: QuickSpec {
                 
                 KakapoServer.get("/comments/:id") { request in
                     usersInfo = request.info
+                    return db.find(CommentFactory.self, id: 2)
                 }
                 
                 KakapoServer.get("/users/:id") { request in
                     usersInfo = request.info
+                    return db.find(UserFactory.self, id: 2)
```
Comment (MP0w) [inline KakapoExample/KakapoTests/KakapoServerTests.swift]: duplicate stuff, why?

File: KakapoExample/KakapoTests/KakapoServerTests.swift [hunk 5]
```diff
@@ -94,13 +104,14 @@ class KakapoServerTests: QuickSpec {
                 KakapoServer.get("/users/:id") { request in
                     // Shouldn't reach here
                     info = request.info
+                    return db.find(CommentFactory.self, id: 1)
```
Comment (MP0w) [inline KakapoExample/KakapoTests/KakapoServerTests.swift]: XCTFail()

**Comment by MP0w** [comment]
closes #14

**Comment by MP0w** [comment]
:+1:

**Comment by joanromano** [comment]
:+1:

**Pull request merged**